hello-libs/libsrc/gmath/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (C) 2010 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+cmake_minimum_required(VERSION 3.4.1)
+
+set(CMAKE_VERBOSE_MAKEFILE on)
+
+add_library(gmath STATIC src/gmath.c)
+
+# copy out the lib binary... need to leave the static lib around to pass gradle check
+set(distribution_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../distribution)
+add_custom_command(TARGET gmath POST_BUILD
+                   COMMAND "${CMAKE_COMMAND}" -E
+                   copy "$<TARGET_FILE:gmath>"
+                   "${distribution_DIR}/gmath/lib/${ANDROID_ABI}/$<TARGET_FILE_NAME:gmath>"
+                   COMMAND "${CMAKE_COMMAND}" -E
+                   copy "${CMAKE_CURRENT_SOURCE_DIR}/src/gmath.h"
+                   "${distribution_DIR}/gmath/include/gmath.h"
+#                   COMMAND "${CMAKE_COMMAND}" -E
+#                   remove_directory "${CMAKE_CURRENT_BINARY_DIR}"
```

Comment:
yeah, I should remove it.  I was thinking to have people to debug it with 2 lines enabled: the issue is that after cmake removes result artifact files, gradle errors out. I think this sample's usage is legal too:
  after cmake builds the libs, cmake copies them out, cmake removes temporarily and final lib file
but gradle does have a say to this too: gradle knows that there are a bunch of libs get built from cmake, gradle wants to make sure that the final results are there.

It could be either ways I guess. So there 2 lines are left in so later we might bug team to add forgiveness inside gradle. The wired thing is that, the other lib is shared lib, it is ok to be removed by cmake, only this static lib